plastron/auth.py

```diff
@@ -64,14 +65,25 @@ class ProvidedJwtTokenAuth(Auth):
     '''Auth for AUTH_TOKEN provided in config'''
     def __init__(self, config: Dict[str, str]):
         super().__init__(config)
+        self.logger = logging.getLogger(type(self).__name__)
         self.jwt_token = config['AUTH_TOKEN']
+        self.batch_jwt_token = config.get('BATCH_MODE', {}).get('PLASTRON_BATCH', None)
```

Comment:
I think a better name for the secondary key would be `AUTH_TOKEN`. It mirrors what is present at the top-level and is more self-documenting than `PLASTRON_BATCH`.